lib/components/result-view/result-view.js

```diff
@@ -139,6 +139,22 @@ type Props = { store: OutputStore, destroy: Function, showResult: boolean };
     );
   }
 
+  scrollToBottom() {
+    if (!this.el) return;
```

Comment:
We should also return for the cases if `this.expanded === true` or `this.props.isPlain === true` since we don't want to scroll in those cases.